drivers/gpio/Kconfig.tt_bh

```
@@ -0,0 +1,8 @@
+# Copyright (c) 2025 Tenstorrent AI ULC
+# SPDX-License-Identifier: Apache-2.0
+
+config GPIO_TT_BH
+	bool "Tenstorrent Blackhole GPIO"
+	default y
```

Don't forget to add the second line
```suggestion
	default y
	depends on DT_HAS_TENSTORRENT_BH_GPIO_ENABLED
```